agent/sshd/cmd.go

```diff
@@ -25,7 +22,7 @@ func newCmd(u *osauth.User, shell, term, host string, command ...string) *exec.C
 		groups = append(groups, uint32(igid))
 	}
 	if len(groups) == 0 {
-		groups = append(groups, uint32(gid))
+		groups = append(groups, uint32(u.GID))
```

Comment:
I believe this commit ought to be squashed on the previous commit otherwise it will fail to build.